package.json

```diff
@@ -43,6 +44,7 @@
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.21.2",
     "jest": "^26.0.1",
-    "prettier": "2.0.5"
+    "prettier": "2.0.5",
```

Comment:
It is unrelated to this PR, but I just noticed that prettier is the only dev dependency that is using an "exact dependency" instead of the "^version" used for the other dependencies (where "^version" means "Compatible with version" and would relax the dependency to also include new patch releases, See https://docs.npmjs.com/files/package.json#dependencies for more information about the meaning of the syntax that can be used to specify the version of the dependencies).

We may want to be consistent and use the "^version" also for prettier, no need to rush on it, I mainly wanted to mention it. If we want to change that, I would suggest to do it in a separate pull request.

Reply:
Yeah, will send a pr :-)